### PR TITLE
Add RTT view to the User page

### PR DESCRIPTION
**Pull request by xiangtin**
## Summary
1. Add RTT view and re-ordering and the User page.
2. Update the 'Host Pool Overview' view to allow switching between Pooled Host Pool and Personal Host Pool, add 'Assigned User' column and remove 'Available Sessions' column to the Personal Host Pool view.

## Validation
https://ms.portal.azure.com/?feature.workbookGalleryBranch=xixiao%2FUserTab

## Screenshots
1. RTT view added to the User page
image

### CODE REVIEW

File: Workbooks/Windows Virtual Desktop/AVD Insights/AVDWorkbookV2.workbook [hunk 1]
```diff
@@ -1152,4 +1152,4 @@
     "Azure Monitor"
   ],
   "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/workbook.json"
-}
+}
```
Comment (mike-mo) [inline Workbooks/Windows Virtual Desktop/AVD Insights/AVDWorkbookV2.workbook:1155]: Revert this file if changes are not intentional

File: Workbooks/Windows Virtual Desktop/shared/Host Pool Overview/Host Pool Overview.workbook [hunk 2]
```diff
@@ -180,11 +180,26 @@
...
+            "resourceType": "microsoft.resourcegraph/resources",
+            "value": null
+          },
+          {
+            "id": "c9f78080-1f60-4900-b137-3b9c319bde41",
+            "version": "KqlParameterItem/1.0",
+            "name": "HostpoolType",
+            "type": 1,
+            "query": "resources\r\n| where type == \"microsoft.desktopvirtualization/hostpools\" and resourceGroup == tolower(\"{ResourceGroup}\")  and id == \"{HostPool}\"\r\n| project hostPoolType= properties.hostPoolType",
+            "crossComponentResources": [
```
Comment (mike-mo) [inline Workbooks/Windows Virtual Desktop/shared/Host Pool Overview/Host Pool Overview.workbook:192]: Does ARG query language support the `=~` operator for case insensitive comparison?

File: Workbooks/Windows Virtual Desktop/shared/Host Pool Overview/Host Pool Overview.workbook [hunk 3]
```diff
@@ -250,7 +265,7 @@
...
...MaxSessionLimit\\\"] -[\\\"Sessions\\\"]\"}},{\"criteriaContext\":{\"operator\":\"Default\",\"rightValType\":\"column\",\"resultValType\":\"static\",\"resultVal\":\"0\"}}]},{\"originalName\":\"[HostUsers].Users\",\"mergedName\":\"Users\",\"fromId\":\"e0297506-948a-4021-b236-175107e751dc\"},{\"originalName\":\"[HostPools].URI\",\"mergedName\":\"URI\",\"fromId\":\"unknown\"},{\"originalName\":\"[HostPools].OSversion\",\"mergedName\":\"OSversion\",\"fromId\":\"unknown\"},{\"originalName\":\"[HostPools].AssignedUser\",\"mergedName\":\"AssignedUser\",\"fromId\":\"unknown\"},{\"originalName\":\"[HostPools].Name\"},{\"originalName\":\"[hostpoolspecifics].resourceGroup\"},{\"originalName\":\"[hostpoolspecifics].HostPool\"},{\"originalName\":\"[hostpoolspecifics].location\"},{\"originalName\":\"[hostpoolspecifics].LoadBalanceType\"},{\"originalName\":\"[hostpoolspecifics].HostPoolType\"},{\"originalName\":\"[HostUsers].Computername\"},{\"originalName\":\"[HostUsers].SessionHostName\"},{\"originalName\":\"[HostUsers].HostPool\"},{\"originalName\":\"[HostPools].LastUpdateStatus\"},{\"originalName\":\"[HostPools].LastUpdateTime\"},{\"originalName\":\"[hostpoolspecifics]. LoadBalanceType\"}]}",
```
Comment (mike-mo) [inline Workbooks/Windows Virtual Desktop/shared/Host Pool Overview/Host Pool Overview.workbook:268]: Let's output the `AssignedUser` column matching the casing of other columns: `Assigned user`

**Review by mike-mo — CHANGES_REQUESTED** [review]

File: Workbooks/Windows Virtual Desktop/AVD Insights/AVDWorkbookV2.workbook (hunk 1 above)
Comment (xiangtin) [reply Workbooks/Windows Virtual Desktop/AVD Insights/AVDWorkbookV2.workbook]: Reverted the changes.

File: Workbooks/Windows Virtual Desktop/shared/Host Pool Overview/Host Pool Overview.workbook (hunk 2 above)
Comment (xiangtin) [reply Workbooks/Windows Virtual Desktop/shared/Host Pool Overview/Host Pool Overview.workbook:192]: Updated to use '=~' operation for case-insensitive comparison.

File: Workbooks/Windows Virtual Desktop/shared/Host Pool Overview/Host Pool Overview.workbook (hunk 3 above)
Comment (xiangtin) [reply Workbooks/Windows Virtual Desktop/shared/Host Pool Overview/Host Pool Overview.workbook:268]: Updated the column name to 'Assigned user'.

File: Workbooks/Windows Virtual Desktop/AtScale/User Report/User Report.workbook [hunk 4]
```diff
@@ -1562,6 +1218,346 @@
...
+                        "groupBy": [
+                          "ClientType"
+                        ],
+                        "expandTopLevel": true,
+                        "finalBy": "ClientVersion"
+                      }
+                    },
+                    "sortBy": []
+                  },
+                  "customWidth": "42",
```
Comment (mike-mo) [inline Workbooks/Windows Virtual Desktop/AtScale/User Report/User Report.workbook:1556]: I think we wanted these (at scale and scoped) to be 100% width, right?

**Review by mike-mo — CHANGES_REQUESTED** [review]

File: Workbooks/Windows Virtual Desktop/AtScale/User Report/User Report.workbook (hunk 4 above)
Comment (xiangtin) [reply Workbooks/Windows Virtual Desktop/AtScale/User Report/User Report.workbook]: Updated. It makes sense to set the width to 100%.

File: Workbooks/Windows Virtual Desktop/shared/Host Pool Overview/Host Pool Overview.workbook (hunk 2 above)
Comment (mike-mo) [reply Workbooks/Windows Virtual Desktop/shared/Host Pool Overview/Host Pool Overview.workbook:192]: But then you don't need the `tolower()` function anymore

**Review by mike-mo — CHANGES_REQUESTED** [review]

**Review by mike-mo — APPROVED** [review]

**Pull request merged**